tests/unit/test_common.cpp

```diff
@@ -512,6 +515,49 @@ TEST_F(SerializerTest, HNSWSerialzer) {
     output.close();
 }
 
+#if HAVE_SVS
+TEST_F(SerializerTest, SVSSerializer) {
+
+    this->file_name = std::string(getenv("ROOT")) + "/tests/unit/bad_index_svs";
+    auto metadata_path = std::filesystem::path(this->file_name) / "metadata";
+
+    // Try to load an index from a directory that doesn't exist.
+    SVSParams params = {
+        .type = VecSimType_FLOAT32,
+        .dim = 1024,
+        .metric = VecSimMetric_L2,
+        .blockSize = 1024,
+        /* SVS-Vamana specifics */
+        .quantBits = VecSimSvsQuant_NONE,
+        .graph_max_degree = 63, // x^2-1 to round the graph block size
+        .construction_window_size = 20,
+        .max_candidate_pool_size = 1024,
+        .prune_to = 60,
+        .use_search_history = VecSimOption_ENABLE,
+    };
+    VecSimParams index_params = {.algo = VecSimAlgo_SVS, .algoParams = {.svsParams = params}};
```

Comment:
why is it required to initialize all these params to test failure?

Reply:
Removed